beancount/projects/export.py

```diff
@@ -1,32 +1,14 @@
 #!/usr/bin/env python3
-"""List assets and join with attributes of account and commodities to a CSV file.
```

Comment:
Oh that's a real bummer. Is there no way to provide the docstring at the top of the file?
I'd really like to keep it there. It's the first thing you see when you open the file.

Reply:
Sure, it is possible. Fixing this I noticed that the help text gets re-wrapped wrong when output by Click.